cookbook/form/unit_testing.rst

```diff
@@ -164,11 +164,11 @@ Adding Custom Extensions
 It often happens that you use some options that are added by
 :doc:`form extensions </cookbook/form/create_form_type_extension>`. One of the
 cases may be the ``ValidatorExtension`` with its ``invalid_message`` option.
-The ``TypeTestCase`` only loads the core form extension, which means an
-"Invalid option" exception will be raised if you try to test a class that
-depends on other extensions. The
-:method:`Symfony\\Component\\Form\\Test\\TypeTestCase::getExtensions` allows you to
-return a list of extensions to register::
+The ``TypeTestCase`` only loads the core form extensions, which means an
```

Comment:
there is only one core extension (https://github.com/symfony/form/blob/master/Forms.php#L121)